jdaviz/core/validunits.py

```diff
@@ -19,68 +18,61 @@ def units_to_strings(unit_list):
     result : list
         A list of the units with their best (i.e., most readable) string version.
     """
-    return [u.Unit(unit).name
-            if u.Unit(unit) == u.Unit("Angstrom")
-            else u.Unit(unit).long_names[0] if (
-                hasattr(u.Unit(unit), "long_names") and len(u.Unit(unit).long_names) > 0)
```

Comment:
Note: Long name does not round-trip well with the display axis stuff. Let's not use it.